harvester/utils.py

```diff
@@ -1,87 +1,106 @@
+import json
+
 import pandas as pd
 import geopandas as gp
+import requests
 from shapely.geometry import Point
 
 
+def folder_creation(path):
```

Comment:
This is minor but I prefer a more imperative function naming schema. `get_something`, `create_folder`. This is totally opinionated